css-four-column-layout/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CSS FOUR COLUMN LAYOUT</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <main class="container">
+    <h2>How It Works</h2>
+    <section class="row">
+      <div class="col-one-fourth">
```

Comment:
make sure there is a class in the CSS which matches `col-one-fourth` and has the appropriate width set on it.

Reply:
Updated.